ffmpeg_normalize/__main__.py

```diff
@@ -249,6 +249,22 @@ def create_parser() -> argparse.ArgumentParser:
         ),
     )
 
+    group_ebu.add_argument(
+        "--safe-ebu",
```

Comment:
I'm not sure if this is the most easily understandable option name. It implies that it is always the safest option … is that the case?

Maybe `--auto-lower-loudness-target` — and then we should just clarify in the README how to use these various options.

Reply:
Making overly verbose options does not make reading the manual unnecessary. It only makes writing and memorizing them harder. Especially when there is no bash completion for it.

Reply:
I understand that reasoning, however I'd like to strike a balance between usability and understandability here.

I personally very much prefer speaking arguments, even if they are longer, and maybe there will be a "one size fits all" option in the future that could carry this name? Or let's call it `-sm / --safe-ebu-margin`?

If completions are an issue, let's create bash/zsh completions and ship them with this repo. I think this should be easy to do …

Reply:
Added some basic completions here: https://github.com/slhck/ffmpeg-normalize/commit/d0d274159bb979004718a6e0ca3d707bf11330bf